### ISSUE TITLE
maximumSize not honored if size is the same number

### ISSUE THREAD

**Issue by brambow** [aeagle/react-spaces#46]
**Describe the bug**
Great library! In a Resizable component (i.e. <LeftResizable>) if the size and maximumSize numbers are the same, maximumSize does not appear to be honored and I can resize beyond the max; however, if I set size to be one below the max then behavior is as expected.

**To Reproduce**
1. Create a resizable component that uses the same value for size and maximumSize

```
      <Space.ViewPort>
        <Space.Top size="60px" style={{ backgroundColor: '#808080' }}>
          This is the header
        </Space.Top>
        <Space.Fill>
          <Space.LeftResizable
            trackSize={true}
            size={350}
            minimumSize={0}
            maximumSize={350}
            style={{
              backgroundColor: 'beige'
            }}
          >
            This is the side Layer List
          </Space.LeftResizable>
          <Space.Fill scrollable>
            Fill Space
          </Space.Fill>
        </Space.Fill>
      </Space.ViewPort>
```

**Expected behavior**
Expect maximumSize to be enforced, but it's not.

**Desktop (please complete the following information):**
 - OS: MacOS 10.13
 - Browser Chrome 78

**Comment by aeagle** [comment]
Hi @brambow.

Thanks for the feedback. I'm preparing a new release which fixes this problem. You can check the behaviour here before release https://codesandbox.io/s/peaceful-rain-19w1b or try the alpha package `v0.1.20-alpha.3`.

**Comment by aeagle** [comment]
Fix released on `0.1.20` now.
